.eslintrc.json

```diff
@@ -23,6 +24,31 @@
         "message": "Files outside of `src/api` must not refer to files in `src/api`. See README.md"
       }]
     }],
-    "import/no-absolute-path":"error"
+    "import/no-absolute-path":"error",
+    "no-restricted-imports": [
+      "error",{
+        "paths":[
+          // files under src
+        "AppLayout",
```

Comment:
why do we need to specify files? I don't think it's a workable solution as we will constantly add and rename files.